cmd/livepeer/starter/starter.go

```diff
@@ -1095,6 +1095,16 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 			return
 		}
 
+		// Get base pixels per unit.
```

Comment:
Moved to ensure that the command line flag is used when the `PixelsPerUnit` is missing in the configuration file.

Reply:
I'm not sure if it's allowed to be missing, but makes sense. Is this how the per gateway configs work as well? (dont need to be the same, just to know if it's a new thing or a fix)

Reply:
The reason I allow the `pixels_per_unit` to be missing in the AI config is that it can fall back to the default value of 1, which is set [here](https://github.com/livepeer/go-livepeer/blob/9cdeeb5a22674e78bd5593f22604858ed6c2feb2/cmd/livepeer/starter/starter.go#L219).

In the current behavior, if `pixels_per_unit` is not set in the config, it defaults to the value specified in the `-pixelsPerUnit` flag, or to 1 if that flag is also missing.

For pricing, the system will first check the `price_per_unit` in the config. If that is not set, it will use the value specified in the `-pricePerUnit` flag for all pipelines.

We could also opt to throw an error if these prices are not set. This could be implemented in a separate pull request. For the `pricePerUnit` this might make sense. I don't have a real preference.